src/components/CurrencyInputPanel/index.tsx

```diff
@@ -265,7 +265,7 @@ export default function CurrencyInputPanel({
           onDismiss={handleDismissSearch}
           onCurrencySelect={onCurrencySelect}
           selectedCurrency={currency}
-          otherSelectedCurrency={otherCurrency}
+          otherSelectedCurrency={new Array(1).fill(otherCurrency)}
```

Comment:
Could we use some better syntax like `[otherCurrency]`?

Reply:
it throws ts error when done like that because it can be undefined...I think this is cleaner in the end